Makefile

```diff
@@ -1,7 +1,7 @@
 all: build
 
 build:
-	@jbuilder build @install @test --dev
+	@jbuilder build @install @test
```

Comment:
`--dev` is now implicit in `dune`.  Will make the explicit switch to dune so you can rebase on it.

Reply:
Yeah I was getting this error with the --dev:

```
dune: --dev is no longer accepted as it is now the default.
Usage: dune build [OPTION]... [TARGET]...
Try `dune build --help' or `dune --help' for more information.
make: *** [Makefile:4: build] Error 1
```

Reply:
I just pushed a full port to Dune (and the included configurator).  Could you rebase on top if it?

Reply:
(Remove your changes to the Makefile and use `dune` instead of `jbuild`.)

Reply:
I just rebased it